README.md

```diff
@@ -1 +1,9 @@
-# argoapp
\ No newline at end of file
+# argoapp
+
+A convenience library managing Giant Swarm Management Cluster Argo CD Application CRs.
```

Comment:
Try to say `Giant Swarm Management Cluster Argo CD Application CRs` 3 times in a row